cdb2api/cdb2api.h

```diff
@@ -201,6 +201,7 @@ int cdb2_clone(cdb2_hndl_tp **hndl, cdb2_hndl_tp *c_hndl);
 int cdb2_next_record(cdb2_hndl_tp *hndl);
 
 int cdb2_get_effects(cdb2_hndl_tp *hndl, cdb2_effects_tp *effects);
+const char *cdb2_get_cost(cdb2_hndl_tp *hndl);
```

Comment:
Maybe just declare it in cdb2sql.cpp. I don't think this function will get a lot of usage except from cdb2sql.